importer.go

```diff
@@ -122,7 +122,11 @@ func (i *Importer) readViaCache(imp string) (jsonnet.Contents, error) {
 		return content, nil
 	}
 
-	content, err := fetch(addScheme(imp), i.fetcher())
+	u, err := addScheme(imp)
+	if err != nil {
+		return noContent, err
+	}
+	content, err := fetch(u, i.fetcher())
```

Comment:
actually, not part of this PR, I think I'd invert the logic just below:
```go
content, err := fetch(u, i.fetcher())
if err != nil {
    return noContent, err
}
i.cache[imp] = content
return content, nil

Reply:
I think you're right. It did catch me out a little when i saw it again. Not sure why I did it this way.